pocs/linux/kernelctf/CVE-2024-50264_lts_cos/exploit/cos-109-17800.309.84/exploit.c

```diff
@@ -0,0 +1,851 @@
+#define _GNU_SOURCE
+#include <stdio.h>
```

Comment:
Most of the files are included twice, some of the three times. Please remove the unnecessary includes.